ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js

```diff
@@ -1952,71 +1954,67 @@
         'Dalamud Dive': 'Dalamud-Sturzflug',
         'Death Sentence': 'Todesurteil',
         'Deathstorm': 'Todessturm',
+        'Dive . Dynamo/Chariot': 'Sturzflug + Dynamo/Streitwagen',
+        'Dive Dynamo Combo': 'Sturzflug Dynamo Kombo',
+        'Divebomb': 'Sturzbombe',
+        'Doom': 'Verhängnis',
+        'Dynamo . Beam/Chariot': 'Dynamo + Strahl/Streitwagen',
         'Earth Shaker': 'Erdstoß',
         'Engage!': 'Start!',
         'Enrage': 'Finalangriff',
         'Exaflare': 'Exaflare',
         'Fellruin Trio': 'Untergangs-Trio',
         'Fireball': 'Feuerball',
-        'Flames Of Rebirth': 'Flammen Der Wiedergeburt',
+        'Fireball Soak': 'Feuerball aufsaugen',
+        'Flames Of Rebirth': 'Flammen der Wiedergeburt',
         'Flare Breath': 'Flare-Atem',
         'Flatten': 'Einebnen',
         'Generate': 'Formung',
         'Gigaflare': 'Gigaflare',
         'Grand Octet': 'Großes Oktett',
         'Hatch': 'Austritt',
-        'Heavensfall(?! )': 'Himmelssturz',
         'Heavensfall Trio': 'Himmelssturz-Trio',
+        'Heavensfall(?! )': 'Himmelssturz',
```

Comment:
Correcting what I said above, about using `^` and `$` above, but I think that's not what we want.  I think something like this negative lookahead (or a negative lookbehind) regex is what we need to add to potentially conflicting replacements.

Reply:
dou you have an example of the lookahead (e.g. for augmented rocket punch)?
Than i could add these to all fights where i know these happend (e.g. run, copy, paste which were not mentioned in the comments so far)